examples/pages/dynamic_layer_control.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+import folium
+import folium.features
+import geopandas as gpd
+import shapely
+import streamlit as st
+
+from streamlit_folium import st_folium
+
+st.set_page_config(layout="wide")
+
+
+def main():
```

Comment:
In general, you don't need to do the `def main()` bit with streamlit

Reply:
Ah yeah, I removed it